admin/zabbix/Makefile

```diff
@@ -75,6 +75,14 @@ define Package/zabbix-server
   DEPENDS += +pgsql-cli +libevent2
 endef
 
+define Package/zabbix-server-frontend
+  $(call Package/zabbix/Default)
+  TITLE+= server-frontend
+  DEPENDS += +zabbix-server +php7 +php7-cgi +php7-fpm +php7-mod-mysqli \
```

Comment:
you are using `php7-cgi` here, so I don't think you need `php7-fpm`
not sure you need `php7` (cli ?)
`zabbix-server` is not a strict requirement
`php7-mod-mysqli` should be selected by db backend (if we build with postgresql support, select the correct backend)

Reply:
Done